### PR TITLE
rust/NFT tut files

### PR DESCRIPTION
**Pull request by myklemykle**
The two files edited in the tutorial.

### CODE REVIEW

File: near/6_NFT/contracts/rust/lib.rs [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+        }
+        // No token with that ID exists, mint and add token to data structures
+        self.token_to_account.insert(&token_id, &owner_id);
+
+        // Generate random Flarn DNA:
+        use rand::prelude::*;
+        use rand_chacha::ChaCha8Rng;
+        use rand_seeder::{Seeder};
+        let mut rng: ChaCha8Rng = Seeder::from(env::random_seed()).make_rng();
+        let new_flarn = Flarn {dna: rng.gen()};
```
Comment (pacmessica) [inline near/6_NFT/contracts/rust/lib.rs:184]:
```suggestion
        let new_flarn = Flarn {dna: rng.gen()};
```

File: near/6_NFT/contracts/rust/lib.rs [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+        // No token with that ID exists, mint and add token to data structures
+        self.token_to_account.insert(&token_id, &owner_id);
+
+        // Generate random Flarn DNA:
+        use rand::prelude::*;
+        use rand_chacha::ChaCha8Rng;
+        use rand_seeder::{Seeder};
+        let mut rng: ChaCha8Rng = Seeder::from(env::random_seed()).make_rng();
+        let new_flarn = Flarn {dna: rng.gen()};  
+        self.token_to_flarn.insert(&token_id, &new_flarn);
```
Comment (pacmessica) [inline near/6_NFT/contracts/rust/lib.rs:185]:
```suggestion
        self.token_to_flarn.insert(&token_id, &new_flarn);
```

**Review by pacmessica — APPROVED** [review]

**Comment by pacmessica** [comment]
lgtm. Not sure about the pathway though since the folders are mostly empty. what do you think @sosedoff ?

**Comment by sosedoff** [comment]
i don't think we're going to use his tutorial in the pathway so we can keep the structure as is. last time i checked we would need to change the package.json commands to accommodate the change in the directory structure.

**Pull request merged**